doc/geoipupdate.md

```diff
@@ -62,15 +62,20 @@ open.
 
 # NOTES
 
-Typically you should run `geoipupdate` weekly. On most Unix-like systems,
-this can be achieved by using cron. Below is a sample crontab file that
-runs `geoipupdate` on each Wednesday at noon:
+Typically you should run `geoipupdate` at least twice a week. Consult
+our
+[database release schedule](https://support.maxmind.com/hc/en-us/articles/4408216129947-Download-and-Update-Databases#h_01G3XX402XKD3J1CMWKNKMDYYZ)
+for more information.
+
+On most Unix-like systems, this can be achieved by using cron. Below is a
```

Comment:
While giving an example crontab is helpful, it is problematic because end users will copy and paste the crontab, resulting in multiple users hitting our servers at the exact same time.

A possible solution is linking to the relevant section of the dev site: https://dev.maxmind.com/geoip/updating-databases#3-run-geoip-update
If you refresh the page, you'll see that the example crontab uses random days of the week and times while still ensuring that the days of the week are spaced out enough to fall within the twice weekly database update.

Reply:
cool, i didn't realize the dev portal did that. updated